inlong-agent/agent-installer/src/main/java/org/apache/inlong/agent/installer/ModuleManager.java

```diff
@@ -106,51 +120,110 @@ public String getCurrentMd5() {
         return currentMd5;
     }
 
+    public ModuleConfig getModule(Integer moduleId) {
+        return currentModules.get(moduleId);
+    }
+
     /**
-     * thread for core thread.
+     * Thread for core thread.
      *
      * @return runnable profile.
      */
     private Runnable coreThread() {
         return () -> {
-            Thread.currentThread().setName("task-manager-core");
+            Thread.currentThread().setName("module-manager-core");
+            restoreFromLocalFile(confPath);
```

Comment:
One thing I am concerned about is whether the result of `restoreFromLocalFile` has effects on `dealWithConfigQueue`.
i.e. does the `dealWithConfigQueue` method be invoked even if the `restoreFromLocalFile` finished in exception?

Reply:
yes, dealWithConfigQueue method WILL be invoked even if the restoreFromLocalFile finished in exception